man/bootctl.xml

```diff
@@ -102,6 +102,17 @@
         <listitem><para>Sets the default boot loader entry. Takes a single boot loader entry ID string as
         argument. The <option>set-oneshot</option> command will set the default entry only for the next boot,
         the <option>set-default</option> will set it persistently for all future boots.</para></listitem>
+
+        <listitem><para>Optionally, the boot loader entry ID may be specified as one of: <option>@default</option>,
+        <option>@oneshot</option> or <option>@current</option>, which correspond to the current default boot loader
+        entry for all future boots, the current default boot loader entry for the next boot, and the currently booted
+        boot loader entry. These special IDs are resolved to the current values of the EFI variables
+        LoaderEntryDefault, LoaderEntryOneShot and LoaderEntrySelected, see <ulink
```

Comment:
enclose the variable names in `<varname></varname>` each?

Reply:
Not fixed yet.

Reply:
oops, didn't push :)